examples/karaf-redis-example/karaf-redis-example-command/pom.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>karaf-redis-example</artifactId>
+        <groupId>org.apache.karaf.examples</groupId>
+        <version>4.2.2-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>karaf-redis-example-command</artifactId>
+
+    <name>Apache Karaf :: Example :: Redis :: Command</name>
+
+    <packaging>bundle</packaging>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.apache.karaf.shell</groupId>
+            <artifactId>org.apache.karaf.shell.core</artifactId>
+            <version>4.2.1</version>
```

Comment:
Please use `${karaf.version}` here.

Reply:
I used `${project.version}` because `${karaf.version}` is not exist in project.